pkg/api/install/install.go

```diff
@@ -89,7 +89,8 @@ func enableVersions(externalVersions []unversioned.GroupVersion) error {
 }
 
 // userResources is a group of resources mostly used by a kubectl user
-var userResources = []string{"rc", "svc", "pods", "pvc"}
+var userResources = []string{"ds", "deploy", "hpa", "ing", "jobs", "petsets",
```

Comment:
would recommend using the full resource names here - I think using the short names was a mistake